omnibus/config/software/erlang-crypto2.rb

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright 2012-2017 Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name "erlang-crypto2"
+default_version "master"
+
+source git: "https://github.com/jaym/erlang-crypto2.git"
+
+# TODO get Jay to convert this to Apache 2
+license "Reserved"
+license_file "LICENSE"
+
+dependency "erlang"
+dependency "rebar"
```

Comment:
This will be rebar 2.  Does this compile with rebar3?  If so, it might be good to make a rebar3 software definition (we can pull their prebuilt binary and copy it into embedded/bin) so we don't have to deal with both rebar 2 and rebar 3 when figuring out build issues in the future.